File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_version-info.ts
// @ts-nocheck
import {VersionInfo as v0_2_0_VersionInfo} from './VersionInfo'
import {VersionInfo as v0_1_0_VersionInfo} from '../v0_1_0/VersionInfo'

export function convert__version_info__from__0_1_0(from: v0_1_0_VersionInfo): v0_2_0_VersionInfo {
    return new v0_2_0_VersionInfo (
        from.current_version,
        from.min_supported_version,
        from.git_sha
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/v0_2_0/from_0_1_0_version_info.rs


pub fn convert__version_info__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::version_info::VersionInfo) -> crate::victron_controller::dashboard::v0_2_0::version_info::VersionInfo {
    crate::victron_controller::dashboard::v0_2_0::version_info::VersionInfo {
        current_version: from.current_version.clone(),
        min_supported_version: from.min_supported_version.clone(),
        git_sha: from.git_sha.clone(),
    }
}